### PR TITLE
Update ESM_DynaMORE_armt.json

### PR DESCRIPTION
**Pull request by JeroenDMW**
questionnaire updated based on feedback from first test round

### CODE REVIEW

File: questionnaires/ESM_DynaMORE/ESM_DynaMORE_armt.json [hunk 1]
```diff
@@ -2070,14 +2074,15 @@
...
-        "branching_logic": "ESM_DynaMORE_physicalContextAndActivityItem1(1) = \"1\" or ESM_DynaMORE_physicalContextAndActivityItem1(2) = \"1\" or ESM_DynaMORE_physicalContextAndActivityItem1(3) = \"1\" or ESM_DynaMORE_physicalContextAndActivityItem1(4) = \"1\" or ESM_DynaMORE_physicalContextAndActivityItem1(5) = \"1\" or ESM_DynaMORE_physicalContextAndActivityItem1(6) = \"1\" or ESM_DynaMORE_physicalContextAndActivityItem1(7) = \"1\" or ESM_DynaMORE_physicalContextAndActivityItem1(8) = \"1\" or ESM_DynaMORE_physicalContextAndActivityItem1(9) = \"1\"",
+        "branching_logic": "ESM_DynaMORE_physicalContextAndActivityItem1(1) = \"1\" or ESM_DynaMORE_physicalContextAndActivityItem1(2) = \"1\" or ESM_DynaMORE_physicalContextAndActivityItem1(3) = \"1\" or ESM_DynaMORE_physicalContextAndActivityItem1(4) = \"1\" or ESM_DynaMORE_physicalContextAndActivityItem1(5) = \"1\" or ESM_DynaMORE_physicalContextAndActivityItem1(6) = \"1\" or ESM_DynaMORE_physicalContextAndActivityItem1(7) = \"1\" or ESM_DynaMORE_physicalContextAndActivityItem1(8) = \"1\" or ESM_DynaMORE_physicalContextAndActivityItem1(9) = \"1\" or
+        ESM_DynaMORE_physicalContextAndActivityItem1(10) = \"1\"",
```
Comment (mpgxvii) [inline questionnaires/ESM_DynaMORE/ESM_DynaMORE_armt.json]: The branching logic must be in one line (2077-2078). This is breaking the questionnaire.

File: questionnaires/ESM_DynaMORE/ESM_DynaMORE_armt.json [hunk 2]
```diff
@@ -2120,21 +2125,22 @@
...
```
Comment (mpgxvii) [inline questionnaires/ESM_DynaMORE/ESM_DynaMORE_armt.json]: The line break here also (2128-2129).

File: questionnaires/ESM_DynaMORE/ESM_DynaMORE_armt.json [hunk 3]
```diff
@@ -2170,14 +2176,15 @@
...
```
Comment (mpgxvii) [inline questionnaires/ESM_DynaMORE/ESM_DynaMORE_armt.json]: The line break here also (2179-2180).

File: questionnaires/ESM_DynaMORE/ESM_DynaMORE_armt.json (hunk 1 above)
Comment (JeroenDMW) [reply questionnaires/ESM_DynaMORE/ESM_DynaMORE_armt.json]: thanks for this, I will remove the unnecessary line breaks

**Comment by mpgxvii** [comment]
LGTM

**Pull request merged**